File: Data Structures & Algorithms/missing-number/submission-1.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int x = 0;
        while(x < nums.size())
        {
            if(nums[x] != x) return x;
            x++;
        }

        return x;
    }
};

File: Data Structures & Algorithms/majority-element/submission-1.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        res = count = 0
        for num in nums:
            if count == 0:
                res = num
            count += (1 if num == res else -1)
        
        return res

"""

Pick a candidate anytime the current count is 0
If the candidate is seen again
    increment
else
    decrement
    

"""